dali/operators/decoder/image_decoder.cc

```diff
@@ -97,15 +97,23 @@ Otherwise, the threads can be reassigned to any CPU core by the operating system
   .AddOptionalArg("split_stages",
       R"code(Applies **only** to the ``mixed`` backend type.
 
-If True, the operator will be split into two sub-stages: a CPU and GPU one.)code",
+If True, the operator will be split into two sub-stages: a CPU and GPU one.
+
+Warning: This argument is now deprecated, and its value will be ignored.
+)code",
       false)
+  .DeprecateArg("split_stages")  // deprecated in DALI 1.0
```

Comment:
I think we could just straight up remove it. If there is a time for a breaking change it is 1.0. And no one uses it anyway.

Reply:
I agree